### PR TITLE
Custom apps update

### PR DESCRIPTION
**Pull request by meganelepalud-akeneo**

### CODE REVIEW

File: content/md/connectivity/how-to-configure-catalogs-for-apps.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+To configure your catalogs, go to the `Connect > Connected Apps`, click on `Manage app`, and then click on `Catalogs`. 
+
+Here you access the list of all the catalogs the app can manage with a label and a status. 
+
+![Catalog list](../img/app-catalog-list.png)
+
+Click on the catalog you want to set up to access the configuration interface. 
+
+When you access a catalog, you will see two tabs:
+- `Settings`: where you can enable or disable a catalog,
```
Comment (tseho) [inline content/md/connectivity/how-to-configure-catalogs-for-apps.md:24]: outdated

File: content/md/connectivity/how-to-configure-catalogs-for-apps.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+- Simple select attribute
+- Text attribute
+- Measurement attribute
+- Yes/No attribute
+- Multi-select attribute
+- Number attribute
+- Text area attribute
+- Date attribute
+- Identifier 
+- Measurement
```
Comment (tseho) [inline content/md/connectivity/how-to-configure-catalogs-for-apps.md:75]: are you sure about documenting all available conditions ? it's easy to forget about it if we add a new one some day.

File: content/md/connectivity/how-to-configure-catalogs-for-apps.md (hunk 1 above)
Comment (meganelepalud-akeneo) [reply content/md/connectivity/how-to-configure-catalogs-for-apps.md]: updated

File: content/md/connectivity/how-to-configure-catalogs-for-apps.md (hunk 2 above)
Comment (meganelepalud-akeneo) [reply content/md/connectivity/how-to-configure-catalogs-for-apps.md:76]: yep! it avoids questions to the support to know why they see or don't see an attribute in the product selection filters.

**Review by tseho — APPROVED** [review]

**Pull request merged**